### PR TITLE
Css vars

### PR DESCRIPTION
**Pull request by munizart**
@danilowoz poderias dar uma olhada nesse PR?
Fiz uns improviments usando [css vars](http://caniuse.com/#feat=css-variables)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-    margin: 0;
-    padding: 0;
+    --yellow: #FFC819;
+    --tomato: #F26A62;
+    --teal: #0EA08A;
+    --selected: rgba(0, 0, 0, .0625);
 }
-
-body {
+:root, body {
```
Comment (danilowoz) [inline css/style.css]: Why this?

File: css/style.css [hunk 2]
```diff
@@ -73,64 +75,67 @@ nav label {
...
 }
 
-#teal:checked~.main .background {
-    transform: translateX(0);
+
+#teal:checked~.main {
+    --offset:  0;
 }
 
 #teal:checked~nav #lbl-teal {
```
Comment (danilowoz) [inline css/style.css]: I think you can rename `--label-color` something like `--label-bg || --select-style` and join the equals selectors, ex: 
```css
    #teal:checked~nav #lbl-teal, 
    #tomato:checked~nav #lbl-tomato, 
    #yellow:checked~nav #lbl-yellow {
        /* */
    }
```

File: css/style.css [hunk 3]
```diff
@@ -1,9 +1,10 @@
 :root {
-    margin: 0;
-    padding: 0;
+    --yellow: #FFC819;
+    --tomato: #F26A62;
+    --teal: #0EA08A;
+    --selected: rgba(0, 0, 0, .0625);
```
Comment (danilowoz) [inline css/style.css]: Choose more functional variable names, ex: `--section-a_color, --selected-color`, so you can to read the variable and understand the value, saca? And not related to your own value, bad examples: `--margin-negative, --button-yellow, --tomato-color`.

File: css/style.css (hunk 1 above)
Comment (munizart) [inline css/style.css]: Lack of sleep, probably

**Comment by munizart** [comment]
Thanks for the review ;D

**Pull request merged**